src/osd/PeeringState.cc

```diff
@@ -257,10 +257,10 @@ void PeeringState::purge_strays()
       psdout(10) << "sending PGRemove to osd." << *p << dendl;
       vector<spg_t> to_remove;
       to_remove.push_back(spg_t(info.pgid.pgid, p->shard));
-      auto m = make_message<MOSDPGRemove>(
+      auto m = TOPNSPC::make_message<MOSDPGRemove>(
```

Comment:
I'm not sure whether the previous commit would build without this one. If not, then need to squash these two.

Reply:
True. Fixed and repushed.